docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -16,21 +16,24 @@ The User Feedback Widget offers many customization options, and if the available
-| `autoInject`   | `boolean`                                  | `true`     | Injects the Feedback widget into the application when the integration is added. Set `autoInject: false` if you want to call `feedback.attachTo()` or `feedback.openDialog()` directly, or only want to show the widget on certain views. |
-| `showBranding` | `boolean`                                  | `true`     | Displays the Sentry logo inside of the form                                                                                                                                                                                              |
-| `colorScheme`  | <code>"system" \| "light" \| "dark"</code> | `"system"` | The color theme to use. `"system"` will follow your OS color scheme.                                                                                                                                                                     |
+| key            | type                                       | default           | description                                                                                                                                                                                                                              |
+| -------------- | ------------------------------------------ | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `autoInject`   | `boolean`                                  | `true`            | Injects the Feedback widget into the application when the integration is added. Set `autoInject: false` if you want to call `feedback.attachTo()` or `feedback.openDialog()` directly, or only want to show the widget on certain views. |
+| `showBranding` | `boolean`                                  | `true`            | Displays the Sentry logo inside of the form                                                                                                                                                                                              |
+| `colorScheme`  | <code>"system" \| "light" \| "dark"</code> | `"system"`        | The color theme to use. `"system"` will follow your OS color scheme.                                                                                                                                                                     |
+| `id`           | `string`                                   | `sentry-feedback` | The `id` attribute of the `<div>` that contains the feedback widget. See [Theme Customization](#theme-customization) for more.                                                                                                           |
```

Comment:
The theme customization doesn't take me anywhere? I think you mean css customization
```suggestion
| `id`           | `string`                                   | `sentry-feedback` | The `id` attribute of the `<div>` that contains the feedback widget. See [CSS Customization](#css-customization) for more.                                                                                                           |
```